common/src/main/resources/tachyon-default.properties

```diff
@@ -132,7 +132,8 @@ tachyon.user.file.writetype.default=MUST_CACHE
 tachyon.user.file.readtype.default=CACHE_PROMOTE
 tachyon.user.heartbeat.interval.ms=1000
 tachyon.user.lineage.enabled=false
+tachyon.user.lineage.master.client.threads=10
 tachyon.user.network.netty.timeout.ms=3000
 tachyon.user.network.netty.worker.threads=0
 tachyon.user.quota.unit.bytes=8MB
-tachyon.user.rawtable.master.client.threads=10
+tachyon.user.raw.table.master.client.threads=10
```

Comment:
What's our convention for property names? We also have names like `file.readtype`.